docs/cli/tkn-results_result_describe.md

```
@@ -0,0 +1,43 @@
+## tkn-results result describe
+
+Describes a Result
+
+```
+tkn-results result describe [-p parent -u uid] [name]
+```
+
+### Examples
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)